.github/workflows/runrms.yml

```diff
@@ -56,4 +55,4 @@ jobs:
 
       - name: Run tests
         if: ${{ always() }}
-        run: pytest -n auto tests -m "not onprem"
+        run: pytest -n auto tests -m "not onprem" --cov --cov-report term-missing
```

Comment:
Will display a box at the end of test output showing what lines of code don't have test coverage